### DISCUSSION TITLE
Where is the GadgetBridge setting?

### DISCUSSION THREAD

**Discussion by cccClyde** [breezy-weather/breezy-weather#778] [Q&A] [answered]
Wanna use it with Smartspacer but can't seem to find the setting according to this guide: https://gadgetbridge.org/basics/integrations/weather/#breezy-weather

**Comment by papjul** [comment]
You need to install Gadgetbridge

**Reply by papjul** [reply, marked as answer]
Starting from v5.1.6, you won't need to install Gadgetbridge, the option will be available if Smartspacer generic weather plugin is installed.

**Reply by pacjo** [reply]
Oh, here it is. I was wondering why support for my app was added out of the blue. Not that I'm not happy. It's awesome to see this.

Not gonna lie, I wasn't expecting to see my plugin mentioned in release notes. Huge thanks for adding support for it.

**Reply by papjul** [reply]
That's my fault, I don't want to show the option to non-users of Gadgetbridge apps because I don't want to promote the purchase of gadgets, so I need to add each new app manually :)

But I'm happy to see that there are app developers that make use of the `WeatherSpec` to do other interesting things! So maybe I should rename the option to something more generic.

That reminds me that I should warn users that any app can intercept that data, including precise longitude and latitude, name of the location and if it is the current location of the user. Especially if I make the option always available, independent of installed apps. I should probably ask Gadgetbridge team if it would be possible to not send location info, and only weather data. I think most users would prefer that way.
